src/style.css

```diff
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+  text-align: center;
+}
+
+body {
+  display: flex;
+  margin: 0;
+}
+
+.wrap {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+.container {
+  display: flex;
+  flex-grow: 1;
+  flex-basis: 33.33%;
+  justify-content: center;
+  align-items: center;
+  --height: 300px;
+  height: var(--height);
+  min-width: var(--height);
+}
+
+.container--1 {
+  order: 1;
+  background-color: rgb(255, 0, 0);
+}
+
+.container--2 {
+  order: 2;
+  background-color: rgb(204, 0, 0);
+}
+
+.container--3 {
+  order: 3;
+  background-color: rgb(153, 0, 0);
+}
+
+.container--4 {
+  order: 4;
+  background-color: rgb(102, 0, 0);
+}
+
+.container--5 {
+  order: 5;
+  background-color: rgb(51, 0, 0);
+}
+
+.container--6 {
+  order: 6;
+  background-color: rgb(0, 0, 0);
+}
```

Comment:
these orders are redundant, too leave it as it is, change them only in media rules